the-collab-lab/tcl-72-smart-shopping-list
# 18. As a user, I want the navbar to be on the top of the page #34

## Description


Changed the look of the navbar

## Related Issue


Closes #34 

## Acceptance Criteria


- Make the Navbar on the top of the page
- Make the title of the page professional
- Make the user indication on the right side of the navbar -> add icons for this as well
- Highlight the current page you are on

## Type of Changes
UI Enhancement

## Updates


### Before
<img width="1440" alt="Screenshot 2024-03-28 at 18 01 10" src="https://github.com/the-collab-lab/tcl-72-smart-shopping-list/assets/103657276/85bb3a6c-c35c-4337-9205-ae0564d6dc37">



### After
<img width="1439" alt="Screenshot 2024-03-28 at 18 00 36" src="https://github.com/the-collab-lab/tcl-72-smart-shopping-list/assets/103657276/32c6c4c0-8cb8-48d6-aec6-95c633518e59">



## Testing Steps / QA Criteria
1. pull the changes from the branch kn-navbar
2. click the sign-in button -> you should see the UI differences
3. click the sign-out button -> you should see the UI differences
4. navigate to different pages by clicking in the navbar -> you should see the page you are on being highlighted
